commands/create.js

```diff
@@ -65,19 +75,58 @@ const filter = m => m.author.id === message.author.id;
         if (reason == 'HIGH_TIME') return message.channel.send('The time cannot be more than 14 days!');
 
         if (reason == 'done' && giveaway.requirements.enabled) {
-            message.channel.send('You can use role requirements: `role=ID`!(without spaces) Once you are finished putting requirements say `done`');
-            const rcollector = message.channel.createMessageCollector(filter, { time: 60 * 1000, max: 1000 });
+            message.channel.send('You can use role requirements: `role=[ID]`!(without spaces), `amari=[Level]` or `wamari=[Weekly amari]`. Once you are finished putting requirements say `done`');
+            const rcollector = message.channel.createMessageCollector({
+                filter,
+                time: 60 * 1000,
+                max: 1000
+            });
             rcollector.on('collect', async (m) => {
-
-                if (!['done', 'stop', 'cancel'].includes(m.content.toLowerCase()) && !m.content.includes('role=')) return rcollector.stop('error');
+                let amari = 0,
+                    wamari = 0
+                if (!['done', 'stop', 'cancel'].includes(m.content.toLowerCase())) return rcollector.stop('error');
```

Comment:
overrides the user input in the collector and returns "You did not provide valid option!"

![image](https://user-images.githubusercontent.com/68866924/135723469-29511d49-1962-4b3d-9721-2c5b434779a0.png)